### ISSUE TITLE
Test failure: SidebarLink

### ISSUE THREAD

**Issue by samajammin** [ethereum/ethereum-org-website#387]
**Describe the bug**

**To Reproduce**

```
~/code/ef/ethereum-org-website « git:(dev)
$ yarn test
yarn run v1.19.1
$ jest
 PASS  docs/.vuepress/theme/components/__tests__/Footer.spec.js
 PASS  docs/.vuepress/theme/components/__tests__/NavLink.spec.js
 PASS  docs/.vuepress/components/__tests__/HomePage.spec.js
 PASS  docs/.vuepress/theme/components/__tests__/DropdownLink.spec.js
 PASS  docs/.vuepress/theme/components/__tests__/Hero.spec.js
 PASS  docs/.vuepress/theme/components/__tests__/SearchBox.spec.js
 FAIL  docs/.vuepress/theme/components/__tests__/SidebarLink.spec.js
  ● Console

    console.error node_modules/vue/dist/vue.runtime.common.dev.js:621
      [Vue warn]: Error in render: "TypeError: (0 , _util.resolveHeaderTitle) is not a function"

      (found in <Root>)
    console.error node_modules/vue/dist/vue.runtime.common.dev.js:1884
      TypeError: (0 , _util.resolveHeaderTitle) is not a function
          at renderLink (/Users/samrichards/code/ef/ethereum-org-website/docs/.vuepress/theme/components/SidebarLink.vue:1080:37)
          at render (/Users/samrichards/code/ef/ethereum-org-website/docs/.vuepress/theme/components/SidebarLink.vue:997:31)
          at createFunctionalComponent (/Users/samrichards/code/ef/ethereum-org-website/node_modules/vue/dist/vue.runtime.common.dev.js:3052:30)
          at createComponent (/Users/samrichards/code/ef/ethereum-org-website/node_modules/vue/dist/vue.runtime.common.dev.js:3225:12)
          at _createElement (/Users/samrichards/code/ef/ethereum-org-website/node_modules/vue/dist/vue.runtime.common.dev.js:3421:13)
          at createElement (/Users/samrichards/code/ef/ethereum-org-website/node_modules/vue/dist/vue.runtime.common.dev.js:3347:10)
          at vm.$createElement (/Users/samrichards/code/ef/ethereum-org-website/node_modules/vue/dist/vue.runtime.common.dev.js:3481:54)
          at Proxy.parentComponentOptions.render (/Users/samrichards/code/ef/ethereum-org-website/node_modules/@vue/test-utils/dist/vue-test-utils.js:4467:12)
          at VueComponent.Vue._render (/Users/samrichards/code/ef/ethereum-org-website/node_modules/vue/dist/vue.runtime.common.dev.js:3532:22)
          at VueComponent.updateComponent (/Users/samrichards/code/ef/ethereum-org-website/node_modules/vue/dist/vue.runtime.common.dev.js:4048:21)
          at Watcher.get (/Users/samrichards/code/ef/ethereum-org-website/node_modules/vue/dist/vue.runtime.common.dev.js:4459:25)
          at new Watcher (/Users/samrichards/code/ef/ethereum-org-website/node_modules/vue/dist/vue.runtime.common.dev.js:4448:12)
          at mountComponent (/Users/samrichards/code/ef/ethereum-org-website/node_modules/vue/dist/vue.runtime.common.dev.js:4055:3)
          at VueComponent.Object.<anonymous>.Vue.$mount (/Users/samrichards/code/ef/ethereum-org-website/node_modules/vue/dist/vue.runtime.common.dev.js:8386:10)
          at mount (/Users/samrichards/code/ef/ethereum-org-website/node_modules/@vue/test-utils/dist/vue-test-utils.js:8649:21)
          at shallowMount (/Users/samrichards/code/ef/ethereum-org-website/node_modules/@vue/test-utils/dist/vue-test-utils.js:8677:10)
          at Object.test (/Users/samrichards/code/ef/ethereum-org-website/docs/.vuepress/theme/components/__tests__/SidebarLink.spec.js:47:21)
          at Object.asyncJestTest (/Users/samrichards/code/ef/ethereum-org-website/node_modules/jest-jasmine2/build/jasmineAsyncInstall.js:102:37)
          at resolve (/Users/samrichards/code/ef/ethereum-org-website/node_modules/jest-jasmine2/build/queueRunner.js:43:12)
          at new Promise (<anonymous>)
          at mapper (/Users/samrichards/code/ef/ethereum-org-website/node_modules/jest-jasmine2/build/queueRunner.js:26:19)
          at promise.then (/Users/samrichards/code/ef/ethereum-org-website/node_modules/jest-jasmine2/build/queueRunner.js:73:41)
          at process._tickCallback (internal/process/next_tick.js:68:7)
    console.error node_modules/vue/dist/vue.runtime.common.dev.js:621
      [Vue warn]: Error in render: "TypeError: (0 , _util.resolveHeaderTitle) is not a function"
```

**Expected behavior**
Tests should pass

**Comment by samajammin** [comment]
Heya @think-in-universe sorry to bother you but thinking this is related to the merging of #364 & #369

Please let me know if you can quickly spot the issue, otherwise, I'll dig into this. Cheers!

**Comment by think-in-universe** [comment]
@samajammin no problem. 

Should be fixed in #390. Tested locally and passed.
